util/src/main/java/com/netcracker/ConsoleHelper.java

```diff
@@ -0,0 +1,39 @@
+package com.netcracker;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class ConsoleHelper {
+    private static final BufferedReader READER = new BufferedReader(new InputStreamReader(System.in));
+
+    private ConsoleHelper(){
+        throw new RuntimeException("Don't create instance of this class");
+    }
+
+    public static void writeMessage(String message){
+        System.out.println(message);
+    }
+
+    public static String readMessage(){
+        while(true) {
+            try {
+                return READER.readLine();
+            } catch (IOException e) {
+                writeMessage("Error. Please try again.");
```

Comment:
It seems weird to print that message to the User, because in case of IOException he will try to re-read anyway and will be blocked in the while look until he succeeds to read something.